apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -56,10 +53,9 @@ export class BookingsController {
   @Post("/reccuring")
   async createReccuringBooking(
     @Req() req: Request & { userId?: number },
-    @Body() _: CreateReccuringBookingInput[],
-    @GetUser() user: User
+    @Body() _: CreateReccuringBookingInput[]
```

Comment:
Can we remove @Body() if its not being used "_" here and in other places in controller?

Reply:
still need to validate the params